apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx

```diff
@@ -0,0 +1,120 @@
+
+We introduced three new features for Supabase Storage: Image resizing, webhooks, and a Smart CDN. These features are designed to work together to deliver a next-gen image resizing system.
+
+- [Blog Post](https://supabase.com/blog/storage-image-resizing-smart-cdn)
+- [Video announcement](https://www.youtube.com/watch?v=NpEl20iuOtg)
+
+## Day 3 - Multi-factor Authentication via Row Level Security Enforcement
+
+![Day 3 - Multi-factor Authentication via Row Level Security Enforcement](/images/blog/2023-01-05-december-beta-update/multi-factor-authentication.png)
+
+We released Multi-factor Authentication (MFA) for everyone, including the option to enforce it via RLS 🔐. Plus, in preparation for releasing SAML, we're "dogfooding" the feature with the introduction of Single Sign On (SSO) on our dashboard.
+
+- [Blog post](https://supabase.com/blog/mfa-auth-via-rls)
+- [Video announcement](https://www.youtube.com/watch?v=He7LI2mv9v0)
+
+## Day 4 - Supabase Wrappers, a Postgres FDW framework
+
+![Day 4 - Supabase Wrappers, a Postgres FDW framework](/images/blog/2023-01-05-december-beta-update/supabase-wrappers.jpg)
+
+Foreign Data Wrappers are a core feature of PostgreSQL. With Supabase Wrappers, we've extended this feature to query other databases or any other external system (including third-party APIs), using SQL.
+
+- [Blog post](https://supabase.com/blog/postgres-foreign-data-wrappers-rust)
+- [Video announcement](https://www.youtube.com/watch?v=QA2qC5F-4OU)
+
+## Day 5 - The Vault is now in Beta
+
+![Day 5 - The Vault is now in Beta](/images/blog/2023-01-05-december-beta-update/vault.jpg)
+
+The Vault is a new Postgres extension that makes it safe and easy to store encrypted secrets and encrypt other stored data in your database.
+
+- [Blog post](https://supabase.com/blog/vault-now-in-beta)
+- [Video announcement](https://www.youtube.com/watch?v=QHLPNDrdN2w)
+
+## Community Day
+
+![Community Day](/images/blog/2023-01-05-december-beta-update/community.jpg)
+
+It's starting to feel like Supabase is really just a "community of communities" 🤗. Far too much has happened in the last 3 months for us to possibly feature it all here, so here's a collection of the highlights!
+
+[Blog post](https://supabase.com/blog/launch-week-6-community-day)
+
+## Postgres Ecosystem
+
+We are all in on Postgres. That is why contributing to the ecosystem is always a priority. Launch Week 6 was no exception, with several announcements:
+
+![Postgres Ecosystem](/images/blog/2023-01-05-december-beta-update/postgres-ecosystem.jpg)
+
+- [pg_graphql v1.0](https://supabase.com/blog/pg-graphql-v1)
+- [PostgREST 11 pre-release](https://supabase.com/blog/postgrest-11-prerelease)
+- [What's new in Postgres 15?](https://supabase.com/blog/new-in-postgres-15)
+- [pg_crdt - an experimental CRDT extension for Postgres](https://supabase.com/blog/postgres-crdt)
```

Comment:
This list doesn't seem to be rendering correctly: https://zone-www-dot-com-git-beta-december-supabase.vercel.app/blog/supabase-beta-december-2022#postgres-ecosystem

Reply:
Seems like the `?` in the link is throwing it off.